README.md

```diff
@@ -2,12 +2,12 @@
 
 Pelikan is a framework for developing cache services. It is:
 
-* **Fast**: Pelikan provides high-throughput and low-latency caching solutions.
+- **Fast**: Pelikan provides high-throughput and low-latency caching solutions.
```

Comment:
oh yeah this is good. I was trying to render the markdown on the CLI and it kept giving me weird formatting, likely because of this